### DISCUSSION TITLE
这个黄泉可以了吗

### DISCUSSION THREAD

**Discussion by ususwcsm** [share121/inter-knot#5464] [崩坏：星穹铁道]
![Screenshot_20240922-000436](https://github.com/user-attachments/assets/855ed242-f80a-4bf1-af31-7720c039e96f)

**Comment by Ayn1631** [comment]
暴击低了吧，有一命吗

**Comment by xintie2023** [comment]
你这不行啊，暴击太低了，有一命的还可以，不过有一命你爆伤就太低了。
